### PR TITLE
Fixes to graphviz related code

### PR DESCRIPTION
**Pull request by erfur**
`d05e816 `: The current block in graphviz output is highlighted even when `graph.gv.current` is `false`. Added a logical check to fix it.
`f5e3f60 `: else if blocks are off by one indentation level

### CODE REVIEW

File: libr/core/canal.c [hunk 1]
```diff
@@ -1311,36 +1311,36 @@ static int core_anal_graph_construct_edges (RCore *core, RAnalFunction *fcn, int
...
-                                                r_cons_printf ("age %s %s\n", from, to);
-                                        } else {
-                                                r_cons_printf ("\t\"0x%08"PFMT64x"\" -> \"0x%08"PFMT64x"\" "
-                                                               "[color=\"%s\"];\n", bbi->addr, bbi->jump,
-                                                               bbi->fail != -1 ? pal_jump : pal_trfa);
-                                                core_anal_color_curr_node (core, bbi);
-                                        }
+                        } else if (!is_json && !is_keva) {
+                                if (is_star) {
+                                        char *from = get_title (bbi->addr);
```
Comment (radare) [inline libr/core/canal.c]: this is leaking you may  free from and to after calling printf() below

File: libr/core/canal.c [hunk 2]
```diff
@@ -1311,36 +1311,36 @@ static int core_anal_graph_construct_edges (RCore *core, RAnalFunction *fcn, int
...
-                                        } else {
-                                                r_cons_printf ("\t\"0x%08"PFMT64x"\" -> \"0x%08"PFMT64x"\" "
-                                                               "[color=\"%s\"];\n", bbi->addr, bbi->fail, pal_fail);
-                                                core_anal_color_curr_node (core, bbi);
-                                        }
+                                                    "  <img class=\"connector-end\" src=\"img/arrow.gif\"/></div>\n",
+                                                    bbi->addr, bbi->fail);
+                        } else if (!is_keva && !is_json) {
+                                if (is_star) {
+                                        char *from = get_title (bbi->addr);
```
Comment (radare) [inline libr/core/canal.c]: same here

File: libr/core/canal.c [hunk 3]
```diff
@@ -1311,36 +1311,36 @@ static int core_anal_graph_construct_edges (RCore *core, RAnalFunction *fcn, int
...
+                                        char *to = get_title (bbi->fail);
+                                        r_cons_printf ("age %s %s\n", from, to);
+                                } else {
+                                        r_cons_printf ("\t\"0x%08"PFMT64x"\" -> \"0x%08"PFMT64x"\" "
```
Comment (radare) [inline libr/core/canal.c]: i would prefer to use spaces rather than tabs

File: libr/core/canal.c (hunk 1 above)
Comment (erfur) [reply libr/core/canal.c]: I just wanted to fix the indentation of this block but sure why not :)

File: libr/core/canal.c (hunk 3 above)
Comment (erfur) [reply libr/core/canal.c]: You mean the `\t` inside the format string?

**Comment by radare** [comment]
yes. we try to avoid tabs in the output of the commands

**Pull request merged**